package me.kvait.exception

class NotFoundExceptionWithText(message: String) : RuntimeException(message)